Drivers/Src/CANbus.c

```diff
@@ -16,22 +37,84 @@ void CANbus_Init(void) {
  * @param   CAN bus line
  * @param   id : CAN id of the message
  * @param 	payload : the data that will be sent.
+ * @param blocking: Whether or not this should be a blocking call
  * @return  0 if data wasn't sent, otherwise it was sent.
  */
-int CANbus_Send(CANId_t id, CANPayload_t payload) {
+ErrorStatus CANbus_Send(CANId_t id, CANPayload_t payload, CAN_blocking_t blocking) {

```

Comment:
I believe CAN_blockin_t is not a defined struct

Reply:
It should be defined in the CANbus.h file